src/agent/onefuzz/Cargo.toml

```diff
@@ -20,6 +20,7 @@ notify = "4.0"
 regex = "1.3"
 reqwest = { version = "0.10", features = ["json", "stream"] }
 sha2 = "0.9"
+url = { version = "2.1.1", features = ["serde"] }
```

Comment:
Is this used in this PR?

Reply:
No.  Incidental commit from a PR than I'd split it off.  Since the other is PR is coming, just going to leave this here.